src/main/java/com/microfocus/application/automation/tools/sv/runner/AbstractSvRunDescriptor.java

```diff
@@ -66,9 +68,18 @@ public SvServerSettingsModel[] getServers() {
         return SvServerSettingsGlobalConfiguration.getInstance().getServers();
     }
 
+    public boolean getHasConfigurePermission() {
+        return Utils.hasCurrentUserConfigurePermission();
+    }
+
     @SuppressWarnings("unused")
     public ListBoxModel doFillServerNameItems() {
         ListBoxModel items = new ListBoxModel();
+
+        // Don't allow non-administer users to list servers
+        if (!Utils.hasCurrentUserConfigurePermission())
+            return items;
+
```

Comment:
This block is unnecessary, since the dropdown control will be generated from config.jelly only if the user has the expected permission.

Reply:
Removed.